src/ec/packages/ec-component-site-header/site-header.html.twig

```diff
@@ -143,9 +151,81 @@
+                  label: _language_selector.overlay.close_label,
+                  variant: 'ghost',
+                  type: 'submit',
+                  icon: {
+                    path: _icon_file_path,
+                    type: 'ui',
+                    name: 'close',
+                    size: 's'
+                  },
+                  icon_position: "after",
+                  extra_classes: 'ecl-language-list__close-button',
+                  extra_attributes: [
+                    { name: 'data-ecl-language-list-close', value: true },
+                  ],
+                } only %}
+              </div>
+
+              <div class="ecl-language-list__title ecl-col-12 ecl-col-lg-8 ecl-offset-lg-2" id="ecl-language-list__title">
+                {% include '../ec-component-icon/icon.html.twig' with {
+                  icon: {
+                    path: _icon_file_path,
+                    type: 'general',
+                    name: 'generic-lang',
+                  },
+                  extra_classes: 'ecl-language-list__title-icon',
+                } only %}
+                {{- _language_selector.overlay.title -}}
+              </div>
+            </div>
+            <div class="ecl-row">
+              {% if _language_selector.overlay.items is not empty %}
+                {% for row in _language_selector.overlay.items|batch(_items_per_columns) %}
+                  <div class="ecl-language-list__column ecl-col-12 ecl-col-lg-4{{ loop.index == 1 ? ' ecl-offset-lg-2' }}">
+                    <ul class="ecl-language-list__list">
+                      {% for column in row %}
+                        <li class="ecl-language-list__item{{ column.is_active ? '  ecl-language-list__item--is-active' }}">
+                          {% include '../ec-component-link/link.html.twig' with { 
+                            link: { 
+                              type: 'standalone', 
+                              label: column.lang, 
+                              path: column.href, 
+                              icon_position: 'after',
+                            }, 
+                            icon: column.is_active ? _item_link_active_icon : {},
+                            extra_classes: 'ecl-language-list__link', 
+                            extra_attributes: [ 
+                              { name: 'lang', value: column.lang }, 
+                              { name: 'hrefLang', value: column.lang },
+                              { name: 'rel', value: 'alternate' },
+                            ],
+                          } only %}
+                        </li>
+                      {% endfor %}
+                    </ul>
+                  </div>
+                {% endfor %}
+              {% endif %}
+            </div>
+          </div>
+        </div>
       </div>
     </div>
     {% include '../ec-component-search-form/search-form.html.twig' with _search_form only %}
```

Comment:
We don't  get the label of the input field, basing on the fix we just made on the search form i would expect the label to be set but hidden through its custom class.